cfgov/jinja2/v1/_includes/organisms/image-text-25-75-group.html

```diff
@@ -1,19 +1,3 @@
-{% if not value %}
-    {% macro _image_text_25_75_markup() %}
-        {% include 'molecules/image-text-25-75.html' %}
-    {% endmacro %}
-    {% do global_dict.update(
-        {
-            'image_text_25_75': {
-                'heading': 'Image Text 25 75 Group Header',
-                'image_texts': [
-                    _image_text_25_75_markup(),
-                    _image_text_25_75_markup()
-                ]
-            }
-        })
-    %}
-{% endif %}
 {% set value = value or global_dict.image_text_25_75 %}
```

Comment:
@kurtw I'm not sure I follow - the section you've commented on is removed in this PR. Is there something else that should be removed?